src/index.ts

```diff
@@ -36,6 +36,12 @@ export * from './types';
 
 export { ConversationStore, MemoryStore } from './conversation-store';
 
+export {
+  CustomRoute,
+  ReceiverRoutes,
+  buildReceiverRoutes,
```

Comment:
For developers that create their own receivers, exposing not only the types but also the builder method would be helpful.